src/lib.rs

```diff
@@ -187,10 +187,14 @@ pub use error::TantivyError as Error;
 
 extern crate census;
 extern crate owned_read;
+extern crate chrono;
```

Comment:
I see you reexport the type. I think it might be useful to reexport the entire crate as tantivy::chrono.